client/state/selectors/is-nav-unification-enabled.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Internal dependencies
+ */
+import { isAutomatticTeamMember } from 'calypso/reader/lib/teams';
+import { getReaderTeams } from 'calypso/state/teams/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
+
+export default ( state ) => {
+	// Disable if explicitly requested by the `?disable-nav-unification` query param.
+	if ( new URL( document.location ).searchParams.has( 'disable-nav-unification' ) ) {
+		return false;
+	}
+
+	// Disabled for Jetpack sites.
+	const siteId = getSelectedSiteId( state );
+	if ( isJetpackSite( state, siteId ) && ! isAtomicSite( state, siteId ) ) {
+		return false;
+	}
+
+	// Enable nav-unification for all a12s.
+	if ( isAutomatticTeamMember( getReaderTeams( state ) ) ) {
+		return true;
+	}
+
+	// Enable for E2E tests checking Nav Unification.
+	if ( process.env.FLAGS === 'nav-unification' ) {
+		return true;
+	}
+
```

Comment:
This one won't work, cause Calypso isn't started with `process.env.FLAGS === 'nav-unification'` in the various environments. We had the chance to start the e2e tests with eg `env FLAGS='nav-unification' ./node_modules/.bin/mocha specs/wp-calypso-seo-preview.js` and in that case a feature flag cookie would be added. 

https://github.com/Automattic/wp-calypso/pull/50144/files

Is there a chance that we check for the existence of that cookie?

Reply:
Fixed in e864495.